lua/plenary/window/border.lua

```diff
@@ -33,7 +34,7 @@ function Border._create_lines(content_win_options, border_win_options)
       if title ~= '' then
         title = string.format(" %s ", title)
       end
-      local title_len = string.len(title)
+      local title_len = strings.strdisplaywidth(title)
```

Comment:
Can you also add truncate :) Thanks

Otherwise good work. I am not sure about the module name (currently also can't think of a better one (i am terrible with names), i will ask tj)

Reply:
Sorry, I'm not sure yet how I can see the title sticking out of the borders. Can you show some code to check it?

Reply:
You can just set one of these https://github.com/nvim-telescope/telescope.nvim/blob/284f38c57539967b25c7d32700acffd46599c49f/lua/telescope/pickers.lua#L56-L58 to `string.rep('a', 200)` and do some testing :) You maybe have to do truncate before `" %s ", title`

Reply:
Got it. Fixed on a83722d. This makes titles be truncated like below.

<img width="243" alt="スクリーンショット 0003-03-24 16 18 25" src="https://user-images.githubusercontent.com/1239245/112270297-d201b680-8cbc-11eb-99f8-242b6d408b0f.png">
<img width="245" alt="スクリーンショット 0003-03-24 16 18 39" src="https://user-images.githubusercontent.com/1239245/112270307-d62dd400-8cbc-11eb-9554-47743ee6a20c.png">

Reply:
<3